tests/keras_tests/function_tests/test_activation_quantization_holder_gptq.py

```diff
@@ -43,26 +51,32 @@ class TestGPTQModelBuilderWithActivationHolder(unittest.TestCase):
     def test_adding_holder_instead_quantize_wrapper(self):
         input_shape = (8, 8, 3)
         gptq_model = self._get_gptq_model(input_shape, basic_model)
-        self.assertTrue(isinstance(gptq_model.layers[3], ActivationQuantizationHolder))
-        self.assertTrue(isinstance(gptq_model.layers[3].activation_holder_quantizer, ActivationPOTInferableQuantizer))
+        self.assertTrue(isinstance(gptq_model.layers[4], ActivationQuantizationHolder))
```

Comment:
How did we end up with only this test that needs index changing?
Let's go over it offline